python/meep.i

```diff
@@ -1029,6 +1029,17 @@ void _get_gradient(PyObject *grad, PyObject *fields_a, PyObject *fields_f, PyObj
 }
 
 
+// typemaps for gaussianbeam
+
+%typecheck(SWIG_TYPECHECK_POINTER, fragment="NumPy_Fragments") std::complex<double> E0[3] {
+    $1 = is_array($input);
+}
+
+%typemap(in) std::complex<double> E0[3] {
```

Comment:
Note that your function definition in `meep.hpp` is `EO` (with the letter):

https://github.com/NanoComp/meep/blob/d8bd779ba90731cb5e3439738322b74244424cbf/src/meep.hpp#L1490-L1491

But your typemap is trying to convert  `E0` (the number)

Reply:
Good catch. Fixed.

Reply:
Make sure you make the change in source.cpp too